src/Provider/IslandoraServiceProvider.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Islandora\Crayfish\Commons\Provider;
+
+use Islandora\Crayfish\Commons\CmdExecuteService;
+use Islandora\Crayfish\Commons\FedoraResourceConverter;
+use Pimple\Container;
+use Pimple\ServiceProviderInterface;
+use Silex\Provider\DoctrineServiceProvider;
+use Silex\Provider\MonologServiceProvider;
+use Silex\Provider\ServiceControllerServiceProvider;
```

Comment:
Silex\Provider\ServiceControllerServiceProvider i not used here. ServiceProviderInterface comes from pimple